testing/mocker/mocker_fixture.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from typing import List
+
+from testing.functional_tests.t_utils import create_tmp_dir
+from testing.mocker.create_derivatives import CreateBIDSDerivatives
+from testing.mocker.create_subjects import CreateBIDSSubjects
+
+def create_mock_bids_file_structures(path_temp):
+    """
+    Nothing strange, subjects, sessions, plans.
+    :return:
+    """
+    # Ensure parent folder exists.
+    create_tmp_dir(copy_data_testing_dir=False)
+
+    path_mock_data = path_temp
+
+    #######################
+    # Subject Mock Section
+    #######################
+
+    list_subjects = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+    list_subject_sessions = [1, 2, 3, 4, 5]
+    # list_subject_sessions = None
+
+    # This must be in the order of appending
+    # MUST have bids_details keyword
+    list_subject_specific_bids_dict = [
+        {
+            "acq": ["MTon", "MToff", "T1w"],  # Acquisition
+            "MODALITY": ["MTS"],  # Modality
+        },
+        {
+            "flip": [1, 2],  # Flip angle
+            "mt": ["on", "off"],  # MT
+            "MODALITY": ["MTS"],  # Modality
+        },
+        {
+            "flip": [2],  # Flip angle
+            "mt": ["off"],  # MT
+            "MODALITY": ["MTS"],  # Modality
```

Comment:
The previous two dicts in the example should cover those scenarios already. Also.. since we technically don't have "contrast" concept anymore at generation time.. all keywords are just combined multiplicatively consecutively. In here the third example, the goal was actually to test single contrast stability in terms of its generation.